docs/docs/rest-api/public/api/v2/types/enrichedTask.raml

```diff
@@ -37,6 +37,11 @@ types:
       version?: string
       localVolumes?: LocalVolumeId[]
 
+  EnrichedTaskSingle:
```

Comment:
do we need the single one? Isn't it enough to keep tasks property, with array and just a single item in it? Then I as a client can use the same deserializer to read it...

Reply:
Yes. The current API has the `task` field for a single task and `tasks` for multiple. We can change this in a future version.

Reply:
oh, ok ... that's unfortunate... I thought that we are already making a breaking change by switching from the previous task serialization to this one so it would be ok to break this as well... (since people will have to change the way they deserialize responses anyway)

Reply:
I submit that we just call this "Task" and "TaskList" (Task list, task being singular, is the common grammar).

`EnrichedTask` is an internal concern that API consumers can safely ignore.

Reply:
I submit that we just call this "Task" and "TaskList" (Task list, task being singular, is the common grammar).

EnrichedTask is an internal concern that API consumers can safely ignore.

Reply:
Agree with @timcharper. `EnrichedTask` is an internal construct meant to wrap information that needs to be serialized; to the user, it's just `Task` information. `Task/TaskList` makes sense to me. I'd love to see the naming adjusted.

Reply:
So this should be `TaskSingle` then, right?